From 8b1751de011873d250cd0b58d9c692b1a8a074a5 Mon Sep 17 00:00:00 2001
From: Julian Scheel <julian@jusst.de>
Date: Fri, 5 Oct 2012 12:34:30 +0200
Subject: [PATCH] Disable Werror for deprecated in libsoup plugin.

The plugin uses deprecated functions in libsoup. As we will switch to a more
recent gstreamer version soon, just disable the fatal warning and keep using
the deprecated functions for now.

Signed-off-by: Julian Scheel <julian@jusst.de>
---
 ext/soup/Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/ext/soup/Makefile.am b/ext/soup/Makefile.am
index 6916b17..7a23a02 100644
--- a/ext/soup/Makefile.am
+++ b/ext/soup/Makefile.am
@@ -2,7 +2,9 @@ plugin_LTLIBRARIES = libgstsouphttpsrc.la
 
 libgstsouphttpsrc_la_SOURCES = gstsouphttpsrc.c gstsouphttpclientsink.c gstsoup.c
 
-libgstsouphttpsrc_la_CFLAGS = $(GST_PLUGINS_BASE_CFLAGS) $(GST_BASE_CFLAGS) $(GST_CFLAGS) $(SOUP_CFLAGS) 
+libgstsouphttpsrc_la_CFLAGS = $(GST_PLUGINS_BASE_CFLAGS) $(GST_BASE_CFLAGS) \
+	$(GST_CFLAGS) $(SOUP_CFLAGS) -Wno-deprecated-declarations
+
 libgstsouphttpsrc_la_LIBADD = $(GST_PLUGINS_BASE_LIBS) -lgsttag-@GST_MAJORMINOR@ $(GST_BASE_LIBS) $(SOUP_LIBS)
 libgstsouphttpsrc_la_LDFLAGS = $(GST_PLUGIN_LDFLAGS)
 libgstsouphttpsrc_la_LIBTOOLFLAGS = --tag=disable-static
-- 
1.7.12.2

